Client/Info.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>API_KEY</key>
-	<string>key</string>
+	<string>$(BRAVE_API_KEY)</string>
```

Comment:
Need to test this for staging, will waiting to merge until then. I believe beta builds need the literal `key` = "key", but will verify, may need to add something to the beta channel, but will let you know.